feat: Set many land update operator

fzavalia opened the pull request.

nachomazzara commented on contracts/Rentals.sol:
```diff
@@ -315,6 +315,28 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver, Ree
         emit OperatorUpdated(_contractAddress, _tokenId, _operator, sender);
     }
 
+    /// @notice Set the operator of various individual LANDs inside an Estate
```
```suggestion
    /// @notice Set the operator of LANDs inside an Estate
```

nachomazzara commented on contracts/Rentals.sol:
```diff
@@ -315,6 +315,28 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver, Ree
 
+    /// @notice Set the operator of various individual LANDs inside an Estate
+    /// @dev Differently from the update operator role of the estate, when the asset is transfered to the rentals contract,
+    /// LAND update operators set previously through the Estate contract will remain. 
```
Maybe something like ```suggestion
    /// LAND update operators can be set to assign granular permissions. LAND update operators will remain if they are inside an Estate when it is transferred. They are only cleared once the LAND is transferred. 
```

nachomazzara reviewed (changes requested):
Just left a few comments!

fzavalia commented on contracts/Rentals.sol:
```diff
@@ -315,6 +315,28 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver, Ree
+    function setManyLandUpdateOperator(
+        address _contractAddress,
+        uint256 _tokenId,
+        uint256[] memory _landTokenIds,
```
Good idea, pushed the solution.

fzavalia commented:
@nachomazzara Should we emit an event when this new method is called?
I think it is unnecessary but we are doing it for the other setUpdateOperator function.
Maybe we should remove the other event.

nachomazzara commented:
> @nachomazzara Should we emit an event when this new method is called? I think it is unnecessary but we are doing it for the other setUpdateOperator function. Maybe we should remove the other event.

We can remove them yes. We can set up alarms and track operator changes by looking through the tx

fzavalia commented:
> > @nachomazzara Should we emit an event when this new method is called? I think it is unnecessary but we are doing it for the other setUpdateOperator function. Maybe we should remove the other event.
> 
> We can remove them yes. We can set up alarms and track operator changes by looking through the tx

Removed the event and added tests to check that the LAND contract is emitting them instead
